main.go

```diff
@@ -569,26 +573,30 @@ func addUserCmd(args []string) {
 
 	var out []string
 	exists := false
+
 	for _, line := range lines {
 		if strings.HasPrefix(line, "#") {
 			out = append(out, line)
 			continue
 		}
 		u, p := Cut(line, ":")
+
+		match := CheckPasswordHash(password, p)
+
 		if u == user {
 			exists = true
-			if p == password {
+			if match == true {
```

Comment:
this can be simplified as `if match {`, or even better: `if CheckPasswordHash(password, p)` directly.

Reply:
Sure, I should have did it that way, let me change it

Reply:
PS maybe apply same principle in 365 et sq